.travis.yml

```diff
@@ -0,0 +1,22 @@
+# Travis CI config file
+language: python
+python:
+    - "2.6"
+    - "2.7"
+
+addons:
+    apt:
+        packages:
+        - root-system
+        - root-plugin-hist-*
+        - libroot-*-dev
+        - libroot-bindings-python5.34
+
+before_script:
+    - .travis/install_cmsgemos.sh
+    # Copy Travis chamber info
+    - cp .travis/chamberInfo.py mapping/chamberInfo.py
+    - source .travis/setupenv.sh
+
+script:
+    - ana_scans.py --scandate=current --anaType=scurve
```

Comment:
As mentioned we should expand to test the full range of scripts.  This here only tests `anaUltraScurve.py` and one piece of `ana_scans.py`.